docs/makerd.md

```diff
@@ -127,15 +130,15 @@ OPTIONS:
             [default: user:password]
 
     -d, --data-directory <DATA_DIRECTORY>
-            Optional DNS data directory. Default value:"~/.coinswap/maker"
+            Optional data directory. Default value: "~/.coinswap/maker"
 
     -h, --help
             Print help information
 
     -r, --ADDRESS:PORT <ADDRESS:PORT>
             Bitcoin Core RPC network address
             
-            [default: 127.0.0.1:38332]
+            [default: 127.0.0.1:48332]
```

Comment:
we should actually change this default to 38332 in the apps. 48332 was for testnet4. We are not using testnet anymore. All values should be changed to 38332.